mmv1/templates/terraform/examples/dataplex_datascan_full_profile.tf.erb

```diff
@@ -28,9 +28,26 @@ resource "google_dataplex_datascan" "<%= ctx[:primary_resource_id] %>" {
     exclude_fields {
       field_names = ["property_type"]
     }
+    post_scan_actions {
+      bigquery_export {
+        results_table = "//bigquery.googleapis.com/projects/<%= ctx[:test_env_vars]['project_name'] %>/datasets/<%= ctx[:vars]['dataset_name'] %>/tables/profile_export"
+      }
+    }
   }
 
   project = "<%= ctx[:test_env_vars]['project_name'] %>"
+
+  depends_on = [
+    google_bigquery_dataset.source
+  ]
+}
+
+resource "google_bigquery_dataset" "source" {
+  dataset_id                  = "dataplex_dataset_%{random_suffix}"
```

Comment:
I think this should be:

```suggestion
  dataset_id                  = "<%= ctx[:vars]['dataset_name'] %>"
```

Reply:
it would be great to make the [`datascan_id` earlier in this example](https://github.com/GoogleCloudPlatform/magic-modules/blob/0a1bf679d8719dd25bb295d71b5e39c3cc91373e/mmv1/templates/terraform/examples/dataplex_datascan_full_profile.tf.erb#L4) also use a var - basically the reason we do it this way is so that the user-facing documentation has friendly names, but the tests use a `tf-test` prefix + random suffix, generated from the same example.

Reply:
Ahh i just realized that I added test to the var name as well, so it says "tf-test-test-datascan%{random_suffix}".

I think it's only for internal code, not public facing, but let me submit a commit to clean it up anyways.